FlightReservation/src/consts/style.js

```diff
@@ -108,6 +112,43 @@ const style = StyleSheet.create({
     color: colors.white,
     alignSelf: 'center',
   },
+
+  //* Spinner Modal *//
+  center_item: {
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  modal_container: {
+    backgroundColor: 'rgba(0,0,0,0.5)',
+    height: '102%',
+    width: '100%',
+    position: 'absolute',
+    zIndex: 1,
+  },
+  modal_background: {
+    backgroundColor: colors.green,
+    height: '20%',
+    width: '40%',
+  },
+  spinner: {
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    // color: colors.white,
```

Comment:
If you do not occupy the code that is commented, remove it.

Reply:
Deleted.